sympy/utilities/lambdify.py

```diff
@@ -117,7 +116,6 @@ def _import(module, reload=False):
     These dictionaries map names of python functions to their equivalent in
     other modules.
     """
-    from sympy.external import import_module
```

Comment:
This should not be removed. See https://travis-ci.org/sympy/sympy/jobs/482750089.
```
__________ sympy/printing/tests/test_tensorflow.py:test_codegen_extra __________
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.4.dev0-py2.7.egg/sympy/utilities/runtests.py", line 1306, in _timeout
    function()
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.4.dev0-py2.7.egg/sympy/printing/tests/test_tensorflow.py", line 118, in test_codegen_extra
    f = lambdify((M, N), cg, 'tensorflow')
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.4.dev0-py2.7.egg/sympy/utilities/lambdify.py", line 402, in lambdify
    buf = _get_namespace(m)
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.4.dev0-py2.7.egg/sympy/utilities/lambdify.py", line 520, in _get_namespace
    _import(m)
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.4.dev0-py2.7.egg/sympy/utilities/lambdify.py", line 137, in _import
    module = eval(import_command)
  File "<string>", line 1, in <module>
NameError: name 'import_module' is not defined
```

Reply:
Ahh, clearly! My bad.

I revert it and add a comment. (Unless there is one which I've missed.)